app/survey/components/graphs/WhyNotFairhold.tsx

```diff
@@ -27,23 +31,34 @@ export const WhyNotFairhold = () => {
   }
   
   return (
-      <SurveyGraphCard title="Why wouldn't you choose Fairhold?">
-          <ResponsiveContainer>
+      <SurveyGraphCard 
+        title="Why <em>not</em> Fairhold?"
```

Comment:
Ah! This is why we need `dangerouslySetInnerHTML`.

I think the issue is just that we have `title` as a `string` type which means we get the literal `<em>` appearing in the title.

If we change the props of `SurveyGraphCard` to something like this...

```ts
type Props = PropsWithChildren<{
  title: ReactNode;
  subtitle?: ReactNode;
}>;
```

...and then just pass in something like this, I think we'll achieve what we're looking for safely.

```suggestion
        title={<>Why <em>not</em> Fairhold?</>}
```

This will work as `ReactNode` is a union which contains the `string` type, so `title` can be a chunk of JSX or just a plain string.